sdks/python/setup.cfg

```diff
@@ -25,3 +25,33 @@ verbosity=2
 # fast_coders module which is not available when running unit tests:
 # fast_coders_test and typecoders_test.
 exclude=fast_coders_test|typecoders_test
+
+# Configuration for code coverage.
+# .coveragerc to control coverage.py
```

Comment:
Is `.coveragerc` a separate file? Or is this section the only configuration?

Reply:
`.coveragerc` is the following section to config coverage.py. Looks like we should remove it to avoid confusion.

Reply:
Do you mean removing this line of comment?

Reply:
yes, ling 30, if it makes confusion. what do you think?

Reply:
OK.